### DISCUSSION TITLE
龙腾猫跃代码意见

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#756] [提问] [answered]
龙腾猫跃，你可以让我们一起写代码好吗，你的更新速度实在太慢，你用的是VScode，那也行啊，SSH可以啊，(如果不行，我可以告诉你怎么白嫖CLion)好多有用的想法你没时间搞，都忽略了...（懒得搞.jpg）强迫症提议就是个好例子...，12.34点位可以和你搞（24.33就没人了...）求求了，实在不行教你怎么白嫖CLion(开源)。

**Comment by Light-Beacon** [comment, marked as answer]
VB.net和CLion有什么关系...
PCL的开发不需要更多人
不过你要**真的**想搞代码你可以**提pr**
有别的关于PCL的新具体意见你可以交issue

哦顺带一提 我回的是VS 不是 VSCode

**Reply by WForst-Breeze** [reply]
这哥们交过一次issue说这事了（） #754 ，他提及的 _强迫症建议_ 是 #727

**Reply by Light-Beacon** [reply]
我的说的issue当然是指实质性具体建议而不是这个（

**Comment by xxtg666** [comment]
没想到你还真来问（
6的

**Reply by Enigma-Soul** [reply]
呵呵
